### PR TITLE
Feature/setup ci

### PR DESCRIPTION
**Pull request by arbulu89**
I have restructured a bit the project to setup the CI.
- Executable added to `bin` (it will be installed in /usr/bin/hanadb_exporter
- Packaging files create
- Documentation updated to match with the changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          - tox
+      after_script:
+          - mv tests/coverage.xml .
+          - ./cc-test-reporter after-build -t coverage.py --exit-code $TRAVIS_TEST_RESULT tests/coverage.xml
+
+    - stage: delivery
+      if: type != pull_request
+      env:
+        - FOLDER=/package
+        - PACKAGE_NAME=python-hanadb_exporter
```
Comment (diegoakechi) [inline .travis.yml]: @arbulu89 As this is a application by itself, we don't need to call the package with the "python" prefix. It is only for python modules. I would call the package hanadb_exporter only.

File: README.md [hunk 2]
```diff
@@ -43,7 +48,9 @@ important items in the configuration file:
 
 2. Start the exporter by running the following command:
 ```
-python app.py
+hanadb_exporter -c config.json
```
Comment (diegoakechi) [inline README.md]: We don't need to do it right now, but we need to create also a Systemd unit file for the exporter. This way, it is easier to admin, and pacemaker can also manage this together with the other cluster resources.

File: .travis.yml (hunk 1 above)
Comment (arbulu89) [reply .travis.yml]: @diegoakechi Ok, I will rename the package to `hanadb_exporter`

File: README.md (hunk 2 above)
Comment (arbulu89) [reply README.md]: Perfect! This will come in a future PR

File: python-hanadb_exporter.spec [hunk 3]
```diff
@@ -21,7 +21,7 @@
 %endif
 
 %{?!python_module:%define python_module() python-%{**} python3-%{**}}
-Name:           python-hanadb_exporter
```
Comment (ayoub-belarbi) [inline python-hanadb_exporter.spec]: I think you forgot to change the spec file name itself, it should be `hanadb_exporter.spec` and not `python-hanadb_exporter.spec`

Comment (arbulu89) [reply python-hanadb_exporter.spec]: True... Let me commit that

**Pull request merged**